src/Features/CSharp/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.cs

```diff
@@ -70,9 +70,31 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                 (token.Parent.IsKind(SyntaxKind.XmlElementEndTag) && token.IsKind(SyntaxKind.GreaterThanToken)) ||
                 (token.Parent.IsKind(SyntaxKind.XmlName) && token.Parent.IsParentKind(SyntaxKind.XmlEmptyElement)))
             {
+                // The user is typing inside an XmlElement
                 if (token.Parent.Parent.Kind() == SyntaxKind.XmlElement)
                 {
-                    items.AddRange(GetNestedTags(span));
+                    // Add corresponding tags except ParamRefTag and TypeParamRefTag
+                    items.AddRange(NestedTagNames.Where(name => name != ParamRefTagName && name != TypeParamRefTagName)
+                                                 .Select(name => GetItem(name, span)));
+
+                    // Add a ParamRefTag for each available parameter
+                    var symbol = declaredSymbol as IMethodSymbol;
```

Comment:
I believe that [this](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Symbols/MemberSymbolExtensions.cs,29) is what you want.  The main thing would be to check the symbol kind before calling instead of casting to `IMethodSymbol`